astroquery/heasarc/core.py

```diff
@@ -188,9 +204,10 @@ def _fallback(self, content):
         colstart = [y for x, y in header.items() if "TBCOL" in x]
         collens = [int(float(y[1:]))
                    for x, y in header.items() if "TFORM" in x]
+
         new_table = []
 
-        old_table = content.split("END")[-1].strip()
+        old_table = content.decode().split("END")[-1].strip()
```

Comment:
`content.decode` should (almost) never be needed; you may want to use `response.text` instead of `response.content` when receiving the web content.

Reply:
ok, makes sense, I tried to avoid decode here and in other places where response.text can be used instead.